src/components/UI/UserIcon.vue

```diff
@@ -3,7 +3,7 @@
     :role="isClickable ? 'button' : 'img'"
     :class="$style.container"
     :style="styles.container"
-    @click.stop="openModal"
+    @click.prevent="openModal"
```

Comment:
```suggestion
    @click.prevent.stop="openModal"
```
stopを消さないでほしいです :pray:

Reply:
重ねがけ出来るんですか！！！！？？？？？？？？重ねがけ出来ると思ってなかった
確かにほかの挙動が狂いそうですね、直します